services/management/dbaas/psmdb_cluster_service.go

```diff
@@ -255,6 +255,32 @@ func (s PSMDBClusterService) RestartPSMDBCluster(ctx context.Context, req *dbaas
 	return &dbaasv1beta1.RestartPSMDBClusterResponse{}, nil
 }
 
+// GetPSMDBClusterResources returns expected resources to be consumed by the cluster.
+func (s PSMDBClusterService) GetPSMDBClusterResources(ctx context.Context, req *dbaasv1beta1.GetPSMDBClusterResourcesRequest) (*dbaasv1beta1.GetPSMDBClusterResourcesResponse, error) {
+	settings, err := models.GetSettings(s.db.Querier)
+	if err != nil {
+		return nil, err
+	}
+
+	clusterSize := int64(req.Params.ClusterSize)
+	memory := req.Params.Replicaset.ComputeResources.MemoryBytes * 2 * clusterSize
+	cpu := int64(req.Params.Replicaset.ComputeResources.CpuM) * 2 * clusterSize
+	disk := req.Params.Replicaset.DiskSize*3 + req.Params.Replicaset.DiskSize*clusterSize
+
```

Comment:
Maybe it's just my instance but cluster of size 4 **or** 5 creates **five rs pods**. Could please confirm this and if it is so, could you take this into consideration? @tiagomotasantos

Reply:
I checked the behaviour on minikube and EKS with different cluster sizes. We create N mongos pods, N rs pods and 3 CFG pods (N being the cluster size that the user asks for). Also this is what we pass on the cluster definition. So if you ask for a cluster with 4 nodes you should get 4 rs pods.
![Screenshot 2021-03-09 at 11 59 33](https://user-images.githubusercontent.com/16366502/110467518-08411280-80cf-11eb-9f94-64ac5f5afeca.png)

Reply:
I tried creating PSMDB cluster with 4 nodes and got 4 rs pods 🤔

Reply:
kubectl get pods
```
example-cfg-0                                        2/2     Running   0          90s
example-cfg-1                                        2/2     Running   0          65s
example-cfg-2                                        2/2     Running   0          48s
example-mongos-69f6687b8f-96nz2                      0/1     Running   0          88s
example-mongos-69f6687b8f-d9bwd                      0/1     Running   0          88s
example-mongos-69f6687b8f-gq7rr                      0/1     Running   0          88s
example-mongos-69f6687b8f-qn94z                      0/1     Running   0          88s
example-rs0-0                                        2/2     Running   0          92s
example-rs0-1                                        2/2     Running   0          72s
example-rs0-2                                        2/2     Running   0          52s
example-rs0-3                                        2/2     Running   0          32s
example-rs0-4                                        1/2     Running   0          15s
```
What version of operators are you running?

Reply:
Operator version is what we have on dbaas-controller: `percona-server-mongodb-operator:1.6.0`

Reply:
Also @jprukner if you create a cluster with 3 nodes do you see 3 rs pods or 4?

Reply:
It's 3 rs pods when i create 3 node clusters. Strange.

Reply:
Whatever, I assume it's some kind of ef up on my side.